dotnet/src/SemanticKernel.Abstractions/AI/TextToImage/ITextToImageService.cs

```diff
@@ -29,4 +30,19 @@ public Task<string> GenerateImageAsync(
         int height,
         Kernel? kernel = null,
         CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// Generate images matching the given text description
+    /// </summary>
+    /// <param name="input">Input text for image generation</param>
+    /// <param name="executionSettings">Text to image execution settings</param>
+    /// <param name="kernel">The <see cref="Kernel"/> containing services, plugins, and other state for use throughout the operation.</param>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
+    /// <returns>Generated image contents</returns>
+    [Experimental("SKEXP0001")]
+    public Task<IReadOnlyList<ImageContent>> GetImageContentsAsync(
```

Comment:
Since it's a breaking change and this interface is experimental, do we need to keep already existing method `GenerateImageAsync`? We can also convert old method to be extension method with the same signature.